packages/@dcl/dcl-rollup/ecs.config.ts

```diff
@@ -71,9 +35,38 @@ const config: RollupOptions = {
       extend: true,
       sourcemap: 'hidden',
       compact: true,
-      plugins: [terser({})]
+      plugins: [terser({ format: { comments: false } })]
     }
+  ],
+  plugins: [
+    resolve({
+      preferBuiltins: false,
+      browser: true
+    }),
+    commonjs({
+      exclude: 'node_modules',
+      ignoreGlobal: true
+    }),
```

Comment:
We shouldn't accept common-js for ECS. Why is this needed? ts-proto?

Reply:
I think we use this for @dcl/ecs-math, and any other module that we could import to use at legacy-ecs and ecs